assets/xml/interface/icon_item_vtx_static.xml

```diff
@@ -0,0 +1,12 @@
+<Root>
+    <File Name="icon_item_vtx_static" Segment="11">
+        <DList Name="gItemNamePanelDL" Offset="0x0"/>
+        <DList Name="gLButtonIconDL" Offset="0x90"/>
```

Comment:
Do you mean "Z button icon" here? I know in OoT (which is based on GameCube debug), it's L and R. But I scoured the MM N64 Pause screen, and I don't see any L buttons. I may be opening a huge can of worms here, because I know the long-term goal of *both* OoT and MM decomp projects is to support both N64 and GameCube ROMs, and they're inevitably going to have different button names for this. But I figured I'd bring it up just in case, because in the ROM we are currently in the process of decompiling, I'm not sure this is right.